File: c/1009.c
#include <stdio.h>

int main()
{
    char name[100];
    fgets(name, 100, stdin);

    double fxd_salary, month_total;
    scanf("%lf", &fxd_salary);
    scanf("%lf", &month_total);

    month_total = month_total * 0.15;

    printf("TOTAL = R$ %0.2lf\n", (fxd_salary+month_total));

	return 0;
}

File: lua/1009.lua
local name = io.read()
local fxd_salary = io.read("*n")
local month_total = io.read("*n")

month_total = month_total * 0.15

print("TOTAL = R$ " .. string.format("%0.2f", (fxd_salary+month_total)))
